jfrog_platform/assets/dashboards/jfrog_artifactory_dashboard.json

```
@@ -1,661 +1,797 @@
 {
-    "author_name": "JFrog Platform",
+    "author_info": {
+        "author_name": "JFrog"
+    },
```

```suggestion
    "author_name": "JFrog",
```

I think you may have an old version of ddev. The export command was updated for this field to use `author_name` instead.